central/deployment/queue/observation_queue.go

```diff
@@ -21,8 +24,10 @@ type DeploymentObservationQueue interface {
 	Pull() *DeploymentObservation
 	Peek() *DeploymentObservation
 	Push(observation *DeploymentObservation)
+	PutBackInObservation(observation *DeploymentObservation)
```

Comment:
These functions are not in use yet? what is the plan?

Reply:
that is used at line 529 in manager_impl.go in the function `processNetworkPolicyUpdate`